rollup.config.mjs

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable unicorn/prevent-abbreviations */
+/* cspell:ignore builtins */
+
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import terser from '@rollup/plugin-terser';
+import typescript from '@rollup/plugin-typescript';
+
+import packageJson from './package.json' assert { type: 'json' };
+
+/**
+ * @type {import('rollup').RollupOptions}
+ */
+const esmConfiguration = {
+  cache: false,
+  external: Object.keys(packageJson.dependencies ?? {}),
+  input: './src/index.ts',
+  output: {
+    dir: './lib/esm',
+    format: 'esm',
+    preserveModules: true,
+    preserveModulesRoot: 'src',
+    sourcemap: true,
+  },
+  plugins: [
+    nodeResolve({ preferBuiltins: true }),
+    terser({ ecma: 2022 }),
```

Comment:
Do we need a minifier? Since this module is not mean for frontend, but on the backend it sometimes nice to hack and debug unminified code.

While npm published size should be negligible

Reply:
changed in [75187fb](https://github.com/markelog/grpc-rich-error-model/pull/4/commits/75187fb66eb6c6cc6a4deb8ad632fe8b79a53b90)